source/CardGames.py

```diff
@@ -138,4 +145,28 @@ def dealCards(self, numCards: int, players: PlayerList, deck: Deck):
     for player in players:
       for _ in range(numCards):
         player.addCard(deck.getCard())
-    return True
\ No newline at end of file
+    return True
+    print(player)
```

Comment:
Make sure to take out this print(player) in the final function if this is for testing purposes